cluster-provision/gocli/opts/cnao/cnao_test.go

```diff
@@ -65,4 +77,23 @@ var _ = Describe("CnaoOpt", func() {
 		Expect(spec).NotTo(HaveKey("multus"))
 		Expect(spec).NotTo(HaveKey("multusDynamicNetworks"))
 	})
+
+	It("should execute create CNAO with dynamic networks controller", func() {
+		skipCR = false
+		dncEnabled = true
+		multusEnabled = true
```

Comment:
This one should be disabled in order for CNAO to deploy Multus. (DNC only works with Multus v4)